src/Apps/Auction/getRedirect.ts

```diff
@@ -6,10 +6,10 @@ export interface Redirect {
   reason: string
 }
 
-export function registerRedirect(
-  sale: routes_RegisterQueryResponse["sale"],
-  me: routes_RegisterQueryResponse["me"]
-): Redirect | null {
+export function registerRedirect({
```

Comment:
Switched to types rather than positional arguments for 2 functions.

Reply:
huh - didn't know you can do type destructing this way, very cool